packages/slate-react/src/components/slate.tsx

```diff
@@ -70,16 +71,20 @@ export const Slate = (props: {
 
   useIsomorphicLayoutEffect(() => {
     const fn = () => setIsFocused(ReactEditor.isFocused(editor))
-    document.addEventListener('focus', fn, true)
-    return () => document.removeEventListener('focus', fn, true)
-  }, [])
-
-  useIsomorphicLayoutEffect(() => {
-    const fn = () => setIsFocused(ReactEditor.isFocused(editor))
-    document.addEventListener('blur', fn, true)
-    return () => {
-      document.removeEventListener('focus', fn, true)
-      document.removeEventListener('blur', fn, true)
+    if (IS_REACT_VERSION_17_OR_ABOVE) {
```

Comment:
My only remark would be to add a little summary of your explanation in the PR here just to make it easier to figure out what this code does in the future

Reply:
Have added a summary of my explanation!